package.json

```diff
@@ -13,7 +13,7 @@
   "author": "Rodolphe Belouin <rodolphe.belouin@gmail.com>",
   "license": "MIT",
   "bin": {
-    "clever": "./bin/clever.js",
+    "clever-test": "./bin/clever.js",
```

Comment:
Ooops

Reply:
One more...
Really sorry for the spam :(

Reply:
This is not spam. This is iterative contribution :wink: which is always welcome.